main.go

```diff
@@ -26,25 +24,28 @@ func (m mockClient) Transfer(from, to, amount, passphrase string) error {
 }
 
 func main() {
-	serverHostPort := client.DefaultNodeHostPort
-	datadir := Getwd()
+	dataDir := getwd()
+
+	grpcServerPort := uint(client.DefaultGRPCPort)
+	grpcServer := client.DefaultGRPCServer
+
+	flag.StringVar(&grpcServer, "grpc-server", grpcServer, "The api 2.0 grpc server")
```

Comment:
Same comment as above, no need to reference API version here

Reply:
done